spotify_api_call.py

```diff
@@ -0,0 +1,79 @@
+import requests
+
+
+def get_token():
+    # TODO: set client credentials as environment variables
+    client_id = '0c5ba66cb29e46e8a415a35c19033bdd'
+    client_secret = '280da5fed992434abc6598b3beadb449'
+    headers = {'Content-Type': 'application/x-www-form-urlencoded'}
+    grant_type = {'grant_type': 'client_credentials', 'client_id': client_id, 'client_secret': client_secret}
+
+    json_res = requests.post('https://accounts.spotify.com/api/token', headers=headers, data=grant_type).json()
+
+    token_type = json_res['token_type']
+    access_token = json_res['access_token']
+
+    token_str = f'{token_type} {access_token}'
+    auth = {'Authorization': token_str}
+
+    return auth
+
+
+def get_artist_id(auth):
+    endpoint = 'https://api.spotify.com/v1/search'
+    artist = input('Enter artist name: ')
+
+    search_query = {'q': artist, 'type': 'artist', 'limit': 3}
+
+    search_response = requests.get(endpoint, params=search_query, headers=auth).json()
+
+    artist_name = search_response['artists']['items'][0]['name']
+    artist_id = search_response['artists']['items'][0]['id']
+
+    print(f'Searched artist is ... "{artist_name}"')
+    # TODO: maybe display more artist info
+
+    return artist_name, artist_id
+
+
+def get_top_tracks_by_artist_id(auth, artist_id):
+    json_res = requests.get(f'https://api.spotify.com/v1/artists/{artist_id}/top-tracks',
+                                          headers=auth).json()
+    tracks = json_res['tracks']
+    track_ids = []
+    for tracks in tracks:
```

Comment:
the number of top tracks seems different depending on artist. I suggest that printing first 3 songs if there's more than 3. I will work on handling exceptions for when there's less than 3 or no existing song from the searched artist as well.